src/modules/apps/uninstall.ts

```diff
@@ -35,16 +32,18 @@ const uninstallApps = async (appsList: string[]): Promise<void> => {
   }
 }
 
-export default async (optionalApp: string, options) => {
-  await validateAppAction('uninstall', optionalApp)
-  const app = optionalApp || (await ManifestEditor.getManifestEditor()).appLocator
-  const appsList = [app, ...parseArgs(options._)]
+export default async (optionalApps: string[], options) => {
+  await validateAppAction('uninstall', optionalApps)
+  const appsList = optionalApps.length > 0 ? optionalApps : [(await ManifestEditor.getManifestEditor()).appLocator]
   const preConfirm = options.y || options.yes
+  let promptAnswer
 
   if (!preConfirm) {
-    await promptAppUninstall(appsList)
+    promptAnswer = await promptAppUninstall(appsList)
```

Comment:
Not necessary, but maybe:
```
if (promptAnswer) {
  return
}
```

Reply:
Originally I tried this, but lint didn't like it :(